index.php

```diff
@@ -27,10 +56,26 @@
         <h1>Capynotasaurus</h1>
     </nav>
 
-    <div class="row row-cols-sm-1 row-cols-md-2 row-cols-xl-3 justify-content-center align-items-center mainPage">
-        <?php
-        echo Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
-        ?>
+    <div class="col">
+        <div class="row justify-content-center">
+            <form class="d-flex flex-row align-items-center" action="">
+                <input name="search" type="search" class="form-control m-2" id="search" placeholder="<?php echo $searchedValue; ?>">
```

Comment:
if I search for something the text stays in the search but if i click search again it doesn't use the text displayed by default in the form.

Reply:
Sorted :)